packages/kbn-grid-layout/tsconfig.json

```diff
@@ -2,12 +2,6 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "target/types",
-    "types": [
```

Comment:
I think this is unnecessary - if we remove it, it's gonna take the types from `tsconfig.base.json`